### PR TITLE
APIS-1658: Updated to latest dependencies

### PR DESCRIPTION
**Pull request by rm-hull**

### CODE REVIEW

File: test/resources/logback-test.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (tom59) [inline test/resources/logback-test.xml]: is this used?

Comment (rm-hull) [inline test/resources/logback-test.xml]: Yes I believe so - the `test/resources/logback.xml` squashes application logging when the tests are running, while the `test/resources/logback-test.xml` quietens down the test framework logging.

Comment (tom59) [inline test/resources/logback-test.xml]: I was thinking that when running tests logback would search for logback-test.xml and if not found fallback to logback.xml but never tested it though

Comment (tom59) [inline test/resources/logback-test.xml]: Just tested and I can see that you will have the same result if you delete test/resources/logback.xml and add in logback-test.xml:
+    <logger name="akka" level="ERROR">
 +        <appender-ref ref="STDOUT"/>
 +    </logger>

**Pull request merged**